packages/cli/config/jest.js

```diff
@@ -271,10 +260,11 @@ async function getRootConfig() {
     }),
   ).then(cs => cs.filter(Boolean));
 
+  const rootBaseConfig = await getProjectConfig(targetPath, coverageConfig);
```

Comment:
This brings in a lot of extra config that we don't really want right? We're really just looking to merge any potential config in the root package.json, and maybe root jest configs too? Main concern is that we wouldn't want the full test config merged in here, with `transform` and all